From ac053d6013f7fab73f66db60462c388bea21b714 Mon Sep 17 00:00:00 2001
Message-Id: <ac053d6013f7fab73f66db60462c388bea21b714.1464449390.git.jen@redhat.com>
In-Reply-To: <c7936395ecf322b3de37662c7c6b772e36866cc7.1464449390.git.jen@redhat.com>
References: <c7936395ecf322b3de37662c7c6b772e36866cc7.1464449390.git.jen@redhat.com>
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 13 May 2016 16:09:29 -0400
Subject: [CHANGE 2/3] qxl: Fix new function name for spice-server library
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1463155769-32352-3-git-send-email-kraxel@redhat.com>
Patchwork-id: 70403
O-Subject: [RHEL-7.3 qemu-kvm PATCH 2/2] qxl: Fix new function name for spice-server library
Bugzilla: 1283198
RH-Acked-by: Marc-André Lureau <mlureau@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

From: Frediano Ziglio <fziglio@redhat.com>

The new spice-server function to limit the number of monitors (0.12.6)
changed while development from spice_qxl_set_monitors_config_limit to
spice_qxl_max_monitors (accepted upstream).
By mistake I post patch with former name.
This patch fix the function name.

Signed-off-by: Frediano Ziglio <fziglio@redhat.com>
Acked-by: Christophe Fergeau <cfergeau@redhat.com>
Acked-by: Martin Kletzander <mkletzan@redhat.com>
Signed-off-by: Marc-André Lureau <marcandre.lureau@redhat.com>
(cherry picked from commit a52b2cbf218d52f9e357961acb271a98a2bdff71)
Signed-off-by: Jeff E. Nelson <jen@redhat.com>
---
 hw/display/qxl.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/hw/display/qxl.c b/hw/display/qxl.c
index 1f6a303..748dfce 100644
--- a/hw/display/qxl.c
+++ b/hw/display/qxl.c
@@ -268,8 +268,7 @@ static void qxl_spice_monitors_config_async(PCIQXLDevice *qxl, int replay)
     } else {
 #if SPICE_SERVER_VERSION >= 0x000c06 /* release 0.12.6 */
         if (qxl->max_outputs) {
-            spice_qxl_set_monitors_config_limit(&qxl->ssd.qxl,
-                                                qxl->max_outputs);
+            spice_qxl_set_max_monitors(&qxl->ssd.qxl, qxl->max_outputs);
         }
 #endif
         qxl->guest_monitors_config = qxl->ram->monitors_config;
-- 
2.5.5

